webapp/src/components/preview-movies/style/row/TopRow.tsx

```diff
@@ -25,27 +25,31 @@ function TopRow({ movieList, isDataLoading }: Props) {
   }
 
   return (
-    <div className={'row'} >
+    <div className={'row'}>
       {
         isDataLoading
-          ? <RowLoading/>
+          ? (
+            <RowLoading/>
+          )
```

Comment:
pas besoin de parenthèse ici ça tiens sur une ligne :

```suggestion
          ? <RowLoading/>
```